### PR TITLE
Bridge refactoring integration

### PR DESCRIPTION
**Pull request by juan-cortes**
### Type

Code Improvement

### Context

https://ledgerhq.atlassian.net/browse/LL-1922

### Parts of the app affected / Test plan

Send flow in general

### CODE REVIEW

**Comment by gre** [comment]
for wording related stuff ok Token. we'll probably either let valentin eventually fix that (in context of coin integration) or someone else, but in any case it's not this PR's scope.

**Comment by gre** [comment]
Screenshot_20191019-130522__01__01
Screenshot_20191019-131400__01__01
Screenshot_20191019-133233__01
Screenshot_20191019-133358__01
Screenshot_20191019-133536__01

File: src/locales/en/common.json [hunk 1]
```diff
@@ -286,6 +288,10 @@
       "title": "Opps, your device should not be in Bootloader mode",
       "description": "Please restart your device or contact us"
     },
+    "UnavailableTezosOriginatedAccountReceive": {
+      "title": "Cannot receive funds on originated account",
+      "description": "If you want to receive funds, please use the parent account"
+    },
```
Comment (gre) [inline src/locales/en/common.json]: we'll need to sync these when we have wording

File: src/locales/en/common.json [hunk 2]
```diff
@@ -603,6 +618,10 @@
       "addTokens": "Add token",
       "howTo": "To add token accounts, you need to <0>receive funds</0> through your <1>Ethereum address</1>."
     },
+    "subaccounts": {
+      "seeMore": "See more Subaccounts",
+      "seeLess": "See less Subaccounts"
```
Comment (gre) [inline src/locales/en/common.json]: not sure about the S uppercase

File: src/locales/en/common.json (hunk 1 above)
Comment (valpinkman) [reply src/locales/en/common.json]: yes

File: src/locales/en/common.json (hunk 2 above)
Comment (valpinkman) [reply src/locales/en/common.json]: will change right now

**Pull request merged**